src/thor/timedep_reverse.cc

```diff
@@ -595,19 +598,14 @@ std::vector<PathInfo> TimeDepReverse::FormPath(GraphReader& graphreader, const u
   LOG_DEBUG("path_cost::" + std::to_string(edgelabels_rev_[dest].cost().cost));
   LOG_DEBUG("path_iterations::" + std::to_string(edgelabels_rev_.size()));
 
-  // Get the transition cost at the last edge of the reverse path
-  Cost tc(edgelabels_rev_[dest].transition_cost(), edgelabels_rev_[dest].transition_secs());
-
-  // Form the reverse path from the destination (true origin) using opposing
-  // edges.
+  // Form the reverse path from the destination (true origin) using opposing edges.
   std::vector<PathInfo> path;
-  Cost cost;
+  Cost cost, tc;
```

Comment:
here we want the first edge to have a turn cost of 0 and then, when we get the next edge we put the turn cost onto the beginning of it, by doing this we match the behavior that happens in the forward path expansions.

Reply:
This is a fairly important distinction for the reverse expansion, maybe add this as a comment in the code?

Reply:
Oh I just saw the comment below. Could the `tc` variable be called something like `previous_turn_cost` or similar to make it obvious to the reader that it's different from forward direction?